### DISCUSSION TITLE
Unbranded / Plain apparel generation (on-model)

### DISCUSSION THREAD

**Discussion by marktmcewan** [lllyasviel/Fooocus#2658] [Q&A]
Does anyone have any tips for generating clothing (on-model) to ensure no branding, logos, patterns etc appear on the clothing? I have been experimenting with "plain, unbranded, Fully colored" etc and continue to get mixed results.

**Comment by ILikeToasters** [comment]
I don't find I get many with logos or branding but you could also remove those most likely with inpainting if you do.

**Reply by marktmcewan** [reply]
Thank you, Im new to this and did'nt know that was a feature. Just watched a good video on it so will give that a try. Thanks, Appreciate your time to answer.
